UTBot CLI throws `InaccessibleObjectException` because `java.lang` is not opened

**Description**

UTBot CLI throws `InaccessibleObjectException` because `java.lang` is not opened
Error report with concrete executions failures is generated

**To Reproduce**

1. Download [utbot-cli-java-main-*** from main](https://github.com/UnitTestBot/UTBotJava/actions/runs/5811246767)
2. Unzip it
3. Set `JAVA_HOME` system variable to JDK-17 directory, include JDK's \bin directory in `Path` systems variable and restart
4. Open Windows Powershell
5. Execute command `java -version`. Check JDK 17 is used
6. Create a directory, put utbot-cli JAR-file into it
7. Create src, build and test directories inside
8. Put a Java-file into src (if class is not in default package, create folders inside src and test)
9. Compile Java-file 

`javac  .\src\org\fuzzing\ListConcurrentModification.java -d .\build`

10. Generate tests for the class:

`java -jar .\utbot-cli-main-2023.8.5040.jar generate  -s .\src\org\fuzzing\ListConcurrentModification.java    -cp .\build\    -o  .\test\org\fuzzing\ListConcurrentModificationTest.java    --generation-timeout 30000   org.fuzzing.ListConcurrentModification
`

**Expected behavior**

No exceptions are expected to be thrown when generating tests.

**Actual behavior**

Several similar InaccessibleObjectExceptions are thrown into console.
Error report with concrete executions failures is generated.

**Screenshots, logs**

~~~java
ConcreteExecutor | executeAsync, response(ERROR)
org.utbot.instrumentation.util.InstrumentedProcessError: Error in the instrumented process |> org.utbot.framework.process.kryo.ReadingFromKryoException: Reading from Kryo exception |> com.esotericsoftware.kryo.kryo5.KryoException: java.lang.reflect.InaccessibleObjectException: Unable to make field private transient java.lang.Class java.lang.StackTraceElement.declaringClassObject accessible: module java.base does not "opens java.lang" to unnamed module @5c18298f
Serialization trace:
stackTrace (org.utbot.framework.process.kryo.ThrowableSerializer$ThrowableModel)
exception (org.utbot.framework.plugin.api.UtImplicitlyThrownException)
result (org.utbot.instrumentation.instrumentation.execution.UtConcreteExecutionResult)
        at com.esotericsoftware.kryo.kryo5.serializers.ReflectField.read(ReflectField.java:145)
        at com.esotericsoftware.kryo.kryo5.serializers.FieldSerializer.read(FieldSerializer.java:129)
        at com.esotericsoftware.kryo.kryo5.Kryo.readObject(Kryo.java:774)
        at org.utbot.framework.process.kryo.ThrowableSerializer.read(ThrowableSerializer.kt:100)
        at org.utbot.framework.process.kryo.ThrowableSerializer.read(ThrowableSerializer.kt:20)
        at com.esotericsoftware.kryo.kryo5.Kryo.readObject(Kryo.java:796)
        at com.esotericsoftware.kryo.kryo5.serializers.ReflectField.read(ReflectField.java:123)
        at com.esotericsoftware.kryo.kryo5.serializers.FieldSerializer.read(FieldSerializer.java:129)
        at com.esotericsoftware.kryo.kryo5.Kryo.readObject(Kryo.java:796)
        at com.esotericsoftware.kryo.kryo5.serializers.ReflectField.read(ReflectField.java:123)
        at com.esotericsoftware.kryo.kryo5.serializers.FieldSerializer.read(FieldSerializer.java:129)
        at com.esotericsoftware.kryo.kryo5.Kryo.readClassAndObject(Kryo.java:877)
        at org.utbot.framework.process.kryo.KryoHelper.readObject(KryoHelper.kt:72)
        at org.utbot.instrumentation.ConcreteExecutor$executeAsync$2$1.invokeSuspend(ConcreteExecutor.kt:232)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
        at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:284)
        at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)
        at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
        at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
        at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
        at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
        at org.utbot.common.ConcurrencyKt.runBlockingWithCancellationPredicate(Concurrency.kt:38)
        at org.utbot.framework.plugin.api.TestCaseGenerator$generate$2$3.invoke(TestCaseGenerator.kt:196)
        at org.utbot.framework.plugin.api.TestCaseGenerator$generate$2$3.invoke(TestCaseGenerator.kt:195)
        at org.utbot.common.ConcurrencyKt.runIgnoringCancellationException(Concurrency.kt:47)
        at org.utbot.framework.plugin.api.TestCaseGenerator.generate(TestCaseGenerator.kt:195)
        at org.utbot.framework.plugin.api.TestCaseGenerator.generate$default(TestCaseGenerator.kt:164)
        at org.utbot.cli.GenerateTestsAbstractCommand.generateTestSets(GenerateTestsAbstractCommand.kt:161)
        at org.utbot.cli.GenerateTestsCommand.run(GenerateTestsCommand.kt:113)
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:204)
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:213)
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:17)
        at com.github.ajalt.clikt.core.CliktCommand.parse(CliktCommand.kt:396)
        at com.github.ajalt.clikt.core.CliktCommand.parse$default(CliktCommand.kt:393)
        at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:411)
        at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:436)
        at org.utbot.cli.ApplicationKt.main(Application.kt:31)
Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make field private transient java.lang.Class java.lang.StackTraceElement.declaringClassObject accessible: module java.base does not "opens java.lang" to unnamed module @5c18298f
        at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:354)
        at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:297)
        at java.base/java.lang.reflect.Field.checkCanSetAccessible(Field.java:178)
        at java.base/java.lang.reflect.Field.setAccessible(Field.java:172)
        at com.esotericsoftware.kryo.kryo5.serializers.CachedFields.addField(CachedFields.java:123)
        at com.esotericsoftware.kryo.kryo5.serializers.CachedFields.rebuild(CachedFields.java:99)
        at com.esotericsoftware.kryo.kryo5.serializers.FieldSerializer.<init>(FieldSerializer.java:82)
        at com.esotericsoftware.kryo.kryo5.SerializerFactory$FieldSerializerFactory.newSerializer(SerializerFactory.java:124)
        at com.esotericsoftware.kryo.kryo5.SerializerFactory$FieldSerializerFactory.newSerializer(SerializerFactory.java:108)
        at com.esotericsoftware.kryo.kryo5.Kryo.newDefaultSerializer(Kryo.java:469)
        at com.esotericsoftware.kryo.kryo5.Kryo.getDefaultSerializer(Kryo.java:454)
        at com.esotericsoftware.kryo.kryo5.util.DefaultClassResolver.registerImplicit(DefaultClassResolver.java:89)
        at com.esotericsoftware.kryo.kryo5.Kryo.getRegistration(Kryo.java:581)
        at com.esotericsoftware.kryo.kryo5.Kryo.getSerializer(Kryo.java:601)
        at com.esotericsoftware.kryo.kryo5.serializers.DefaultArraySerializers$ObjectArraySerializer.read(DefaultArraySerializers.java:346)
        at com.esotericsoftware.kryo.kryo5.serializers.DefaultArraySerializers$ObjectArraySerializer.read(DefaultArraySerializers.java:299)
        at com.esotericsoftware.kryo.kryo5.Kryo.readObjectOrNull(Kryo.java:847)
        at com.esotericsoftware.kryo.kryo5.serializers.ReflectField.read(ReflectField.java:132)
        ... 37 more

        at org.utbot.framework.process.kryo.KryoHelper.readObject(KryoHelper.kt:74)
        at org.utbot.instrumentation.ConcreteExecutor$executeAsync$2$1.invokeSuspend(ConcreteExecutor.kt:232)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
        at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:284)
        at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)
        at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
        at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
        at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
        at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
        at org.utbot.common.ConcurrencyKt.runBlockingWithCancellationPredicate(Concurrency.kt:38)
        at org.utbot.framework.plugin.api.TestCaseGenerator$generate$2$3.invoke(TestCaseGenerator.kt:196)
        at org.utbot.framework.plugin.api.TestCaseGenerator$generate$2$3.invoke(TestCaseGenerator.kt:195)
        at org.utbot.common.ConcurrencyKt.runIgnoringCancellationException(Concurrency.kt:47)
        at org.utbot.framework.plugin.api.TestCaseGenerator.generate(TestCaseGenerator.kt:195)
        at org.utbot.framework.plugin.api.TestCaseGenerator.generate$default(TestCaseGenerator.kt:164)
        at org.utbot.cli.GenerateTestsAbstractCommand.generateTestSets(GenerateTestsAbstractCommand.kt:161)
        at org.utbot.cli.GenerateTestsCommand.run(GenerateTestsCommand.kt:113)
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:204)
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:213)
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:17)
        at com.github.ajalt.clikt.core.CliktCommand.parse(CliktCommand.kt:396)
        at com.github.ajalt.clikt.core.CliktCommand.parse$default(CliktCommand.kt:393)
        at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:411)
        at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:436)
        at org.utbot.cli.ApplicationKt.main(Application.kt:31)
Caused by: com.esotericsoftware.kryo.kryo5.KryoException: java.lang.reflect.InaccessibleObjectException: Unable to make field private transient java.lang.Class java.lang.StackTraceElement.declaringClassObject accessible: module java.base does not "opens java.lang" to unnamed module @5c18298f
Serialization trace:
stackTrace (org.utbot.framework.process.kryo.ThrowableSerializer$ThrowableModel)
exception (org.utbot.framework.plugin.api.UtImplicitlyThrownException)
result (org.utbot.instrumentation.instrumentation.execution.UtConcreteExecutionResult)
        at com.esotericsoftware.kryo.kryo5.serializers.ReflectField.read(ReflectField.java:145)
        at com.esotericsoftware.kryo.kryo5.serializers.FieldSerializer.read(FieldSerializer.java:129)
        at com.esotericsoftware.kryo.kryo5.Kryo.readObject(Kryo.java:774)
        at org.utbot.framework.process.kryo.ThrowableSerializer.read(ThrowableSerializer.kt:100)
        at org.utbot.framework.process.kryo.ThrowableSerializer.read(ThrowableSerializer.kt:20)
        at com.esotericsoftware.kryo.kryo5.Kryo.readObject(Kryo.java:796)
        at com.esotericsoftware.kryo.kryo5.serializers.ReflectField.read(ReflectField.java:123)
        at com.esotericsoftware.kryo.kryo5.serializers.FieldSerializer.read(FieldSerializer.java:129)
        at com.esotericsoftware.kryo.kryo5.Kryo.readObject(Kryo.java:796)
        at com.esotericsoftware.kryo.kryo5.serializers.ReflectField.read(ReflectField.java:123)
        at com.esotericsoftware.kryo.kryo5.serializers.FieldSerializer.read(FieldSerializer.java:129)
        at com.esotericsoftware.kryo.kryo5.Kryo.readClassAndObject(Kryo.java:877)
        at org.utbot.framework.process.kryo.KryoHelper.readObject(KryoHelper.kt:72)
        ... 25 more
Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make field private transient java.lang.Class java.lang.StackTraceElement.declaringClassObject accessible: module java.base does not "opens java.lang" to unnamed module @5c18298f
        at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:354)
        at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:297)
        at java.base/java.lang.reflect.Field.checkCanSetAccessible(Field.java:178)
        at java.base/java.lang.reflect.Field.setAccessible(Field.java:172)
        at com.esotericsoftware.kryo.kryo5.serializers.CachedFields.addField(CachedFields.java:123)
        at com.esotericsoftware.kryo.kryo5.serializers.CachedFields.rebuild(CachedFields.java:99)
        at com.esotericsoftware.kryo.kryo5.serializers.FieldSerializer.<init>(FieldSerializer.java:82)
        at com.esotericsoftware.kryo.kryo5.SerializerFactory$FieldSerializerFactory.newSerializer(SerializerFactory.java:124)
        at com.esotericsoftware.kryo.kryo5.SerializerFactory$FieldSerializerFactory.newSerializer(SerializerFactory.java:108)
        at com.esotericsoftware.kryo.kryo5.Kryo.newDefaultSerializer(Kryo.java:469)
        at com.esotericsoftware.kryo.kryo5.Kryo.getDefaultSerializer(Kryo.java:454)
        at com.esotericsoftware.kryo.kryo5.util.DefaultClassResolver.registerImplicit(DefaultClassResolver.java:89)
        at com.esotericsoftware.kryo.kryo5.Kryo.getRegistration(Kryo.java:581)
        at com.esotericsoftware.kryo.kryo5.Kryo.getSerializer(Kryo.java:601)
        at com.esotericsoftware.kryo.kryo5.serializers.DefaultArraySerializers$ObjectArraySerializer.read(DefaultArraySerializers.java:346)
        at com.esotericsoftware.kryo.kryo5.serializers.DefaultArraySerializers$ObjectArraySerializer.read(DefaultArraySerializers.java:299)
        at com.esotericsoftware.kryo.kryo5.Kryo.readObjectOrNull(Kryo.java:847)
        at com.esotericsoftware.kryo.kryo5.serializers.ReflectField.read(ReflectField.java:132)
        ... 37 more

        at org.utbot.instrumentation.ConcreteExecutor.withProcess(ConcreteExecutor.kt:213) ~[utbot-cli-main-2023.8.5040.jar:?]
        at org.utbot.instrumentation.ConcreteExecutor$withProcess$1.invokeSuspend(ConcreteExecutor.kt) ~[utbot-cli-main-2023.8.5040.jar:?]
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [utbot-cli-main-2023.8.5040.jar:?]
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) [utbot-cli-main-2023.8.5040.jar:?]
        at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:284) [utbot-cli-main-2023.8.5040.jar:?]
        at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85) [utbot-cli-main-2023.8.5040.jar:?]
        at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59) [utbot-cli-main-2023.8.5040.jar:?]
        at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source) [utbot-cli-main-2023.8.5040.jar:?]
        at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38) [utbot-cli-main-2023.8.5040.jar:?]
        at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source) [utbot-cli-main-2023.8.5040.jar:?]
        at org.utbot.common.ConcurrencyKt.runBlockingWithCancellationPredicate(Concurrency.kt:38) [utbot-cli-main-2023.8.5040.jar:?]
        at org.utbot.framework.plugin.api.TestCaseGenerator$generate$2$3.invoke(TestCaseGenerator.kt:196) [utbot-cli-main-2023.8.5040.jar:?]
        at org.utbot.framework.plugin.api.TestCaseGenerator$generate$2$3.invoke(TestCaseGenerator.kt:195) [utbot-cli-main-2023.8.5040.jar:?]
        at org.utbot.common.ConcurrencyKt.runIgnoringCancellationException(Concurrency.kt:47) [utbot-cli-main-2023.8.5040.jar:?]
        at org.utbot.framework.plugin.api.TestCaseGenerator.generate(TestCaseGenerator.kt:195) [utbot-cli-main-2023.8.5040.jar:?]
        at org.utbot.framework.plugin.api.TestCaseGenerator.generate$default(TestCaseGenerator.kt:164) [utbot-cli-main-2023.8.5040.jar:?]
        at org.utbot.cli.GenerateTestsAbstractCommand.generateTestSets(GenerateTestsAbstractCommand.kt:161) [utbot-cli-main-2023.8.5040.jar:?]
        at org.utbot.cli.GenerateTestsCommand.run(GenerateTestsCommand.kt:113) [utbot-cli-main-2023.8.5040.jar:?]
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:204) [utbot-cli-main-2023.8.5040.jar:?]
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:213) [utbot-cli-main-2023.8.5040.jar:?]
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:17) [utbot-cli-main-2023.8.5040.jar:?]
        at com.github.ajalt.clikt.core.CliktCommand.parse(CliktCommand.kt:396) [utbot-cli-main-2023.8.5040.jar:?]
        at com.github.ajalt.clikt.core.CliktCommand.parse$default(CliktCommand.kt:393) [utbot-cli-main-2023.8.5040.jar:?]
        at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:411) [utbot-cli-main-2023.8.5040.jar:?]
        at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:436) [utbot-cli-main-2023.8.5040.jar:?]
        at org.utbot.cli.ApplicationKt.main(Application.kt:31) [utbot-cli-main-2023.8.5040.jar:?]
Caused by: org.utbot.framework.process.kryo.ReadingFromKryoException: Reading from Kryo exception |> com.esotericsoftware.kryo.kryo5.KryoException: java.lang.reflect.InaccessibleObjectException: Unable to make field private transient java.lang.Class java.lang.StackTraceElement.declaringClassObject accessible: module java.base does not "opens java.lang" to unnamed module @5c18298f
Serialization trace:
stackTrace (org.utbot.framework.process.kryo.ThrowableSerializer$ThrowableModel)
exception (org.utbot.framework.plugin.api.UtImplicitlyThrownException)
result (org.utbot.instrumentation.instrumentation.execution.UtConcreteExecutionResult)
        at com.esotericsoftware.kryo.kryo5.serializers.ReflectField.read(ReflectField.java:145)
        at com.esotericsoftware.kryo.kryo5.serializers.FieldSerializer.read(FieldSerializer.java:129)
        at com.esotericsoftware.kryo.kryo5.Kryo.readObject(Kryo.java:774)
        at org.utbot.framework.process.kryo.ThrowableSerializer.read(ThrowableSerializer.kt:100)
        at org.utbot.framework.process.kryo.ThrowableSerializer.read(ThrowableSerializer.kt:20)
        at com.esotericsoftware.kryo.kryo5.Kryo.readObject(Kryo.java:796)
        at com.esotericsoftware.kryo.kryo5.serializers.ReflectField.read(ReflectField.java:123)
        at com.esotericsoftware.kryo.kryo5.serializers.FieldSerializer.read(FieldSerializer.java:129)
        at com.esotericsoftware.kryo.kryo5.Kryo.readObject(Kryo.java:796)
        at com.esotericsoftware.kryo.kryo5.serializers.ReflectField.read(ReflectField.java:123)
        at com.esotericsoftware.kryo.kryo5.serializers.FieldSerializer.read(FieldSerializer.java:129)
        at com.esotericsoftware.kryo.kryo5.Kryo.readClassAndObject(Kryo.java:877)
        at org.utbot.framework.process.kryo.KryoHelper.readObject(KryoHelper.kt:72)
        at org.utbot.instrumentation.ConcreteExecutor$executeAsync$2$1.invokeSuspend(ConcreteExecutor.kt:232)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
        at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:284)
        at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)
        at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
        at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
        at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
        at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
        at org.utbot.common.ConcurrencyKt.runBlockingWithCancellationPredicate(Concurrency.kt:38)
        at org.utbot.framework.plugin.api.TestCaseGenerator$generate$2$3.invoke(TestCaseGenerator.kt:196)
        at org.utbot.framework.plugin.api.TestCaseGenerator$generate$2$3.invoke(TestCaseGenerator.kt:195)
        at org.utbot.common.ConcurrencyKt.runIgnoringCancellationException(Concurrency.kt:47)
        at org.utbot.framework.plugin.api.TestCaseGenerator.generate(TestCaseGenerator.kt:195)
        at org.utbot.framework.plugin.api.TestCaseGenerator.generate$default(TestCaseGenerator.kt:164)
        at org.utbot.cli.GenerateTestsAbstractCommand.generateTestSets(GenerateTestsAbstractCommand.kt:161)
        at org.utbot.cli.GenerateTestsCommand.run(GenerateTestsCommand.kt:113)
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:204)
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:213)
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:17)
        at com.github.ajalt.clikt.core.CliktCommand.parse(CliktCommand.kt:396)
        at com.github.ajalt.clikt.core.CliktCommand.parse$default(CliktCommand.kt:393)
        at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:411)
        at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:436)
        at org.utbot.cli.ApplicationKt.main(Application.kt:31)
Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make field private transient java.lang.Class java.lang.StackTraceElement.declaringClassObject accessible: module java.base does not "opens java.lang" to unnamed module @5c18298f
        at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:354)
        at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:297)
        at java.base/java.lang.reflect.Field.checkCanSetAccessible(Field.java:178)
        at java.base/java.lang.reflect.Field.setAccessible(Field.java:172)
        at com.esotericsoftware.kryo.kryo5.serializers.CachedFields.addField(CachedFields.java:123)
        at com.esotericsoftware.kryo.kryo5.serializers.CachedFields.rebuild(CachedFields.java:99)
        at com.esotericsoftware.kryo.kryo5.serializers.FieldSerializer.<init>(FieldSerializer.java:82)
        at com.esotericsoftware.kryo.kryo5.SerializerFactory$FieldSerializerFactory.newSerializer(SerializerFactory.java:124)
        at com.esotericsoftware.kryo.kryo5.SerializerFactory$FieldSerializerFactory.newSerializer(SerializerFactory.java:108)
        at com.esotericsoftware.kryo.kryo5.Kryo.newDefaultSerializer(Kryo.java:469)
        at com.esotericsoftware.kryo.kryo5.Kryo.getDefaultSerializer(Kryo.java:454)
        at com.esotericsoftware.kryo.kryo5.util.DefaultClassResolver.registerImplicit(DefaultClassResolver.java:89)
        at com.esotericsoftware.kryo.kryo5.Kryo.getRegistration(Kryo.java:581)
        at com.esotericsoftware.kryo.kryo5.Kryo.getSerializer(Kryo.java:601)
        at com.esotericsoftware.kryo.kryo5.serializers.DefaultArraySerializers$ObjectArraySerializer.read(DefaultArraySerializers.java:346)
        at com.esotericsoftware.kryo.kryo5.serializers.DefaultArraySerializers$ObjectArraySerializer.read(DefaultArraySerializers.java:299)
        at com.esotericsoftware.kryo.kryo5.Kryo.readObjectOrNull(Kryo.java:847)
        at com.esotericsoftware.kryo.kryo5.serializers.ReflectField.read(ReflectField.java:132)
        ... 37 more

        at org.utbot.framework.process.kryo.KryoHelper.readObject(KryoHelper.kt:74) ~[utbot-cli-main-2023.8.5040.jar:?]
        at org.utbot.instrumentation.ConcreteExecutor$executeAsync$2$1.invokeSuspend(ConcreteExecutor.kt:232) ~[utbot-cli-main-2023.8.5040.jar:?]
        ... 24 more
Caused by: com.esotericsoftware.kryo.kryo5.KryoException: java.lang.reflect.InaccessibleObjectException: Unable to make field private transient java.lang.Class java.lang.StackTraceElement.declaringClassObject accessible: module java.base does not "opens java.lang" to unnamed module @5c18298f
Serialization trace:
stackTrace (org.utbot.framework.process.kryo.ThrowableSerializer$ThrowableModel)
exception (org.utbot.framework.plugin.api.UtImplicitlyThrownException)
result (org.utbot.instrumentation.instrumentation.execution.UtConcreteExecutionResult)
        at com.esotericsoftware.kryo.kryo5.serializers.ReflectField.read(ReflectField.java:145) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.serializers.FieldSerializer.read(FieldSerializer.java:129) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.Kryo.readObject(Kryo.java:774) ~[utbot-cli-main-2023.8.5040.jar:?]
        at org.utbot.framework.process.kryo.ThrowableSerializer.read(ThrowableSerializer.kt:100) ~[utbot-cli-main-2023.8.5040.jar:?]
        at org.utbot.framework.process.kryo.ThrowableSerializer.read(ThrowableSerializer.kt:20) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.Kryo.readObject(Kryo.java:796) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.serializers.ReflectField.read(ReflectField.java:123) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.serializers.FieldSerializer.read(FieldSerializer.java:129) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.Kryo.readObject(Kryo.java:796) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.serializers.ReflectField.read(ReflectField.java:123) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.serializers.FieldSerializer.read(FieldSerializer.java:129) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.Kryo.readClassAndObject(Kryo.java:877) ~[utbot-cli-main-2023.8.5040.jar:?]
        at org.utbot.framework.process.kryo.KryoHelper.readObject(KryoHelper.kt:72) ~[utbot-cli-main-2023.8.5040.jar:?]
        at org.utbot.instrumentation.ConcreteExecutor$executeAsync$2$1.invokeSuspend(ConcreteExecutor.kt:232) ~[utbot-cli-main-2023.8.5040.jar:?]
        ... 24 more
Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make field private transient java.lang.Class java.lang.StackTraceElement.declaringClassObject accessible: module java.base does not "opens java.lang" to unnamed module @5c18298f
        at java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:354) ~[?:?]
        at java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:297) ~[?:?]
        at java.lang.reflect.Field.checkCanSetAccessible(Field.java:178) ~[?:?]
        at java.lang.reflect.Field.setAccessible(Field.java:172) ~[?:?]
        at com.esotericsoftware.kryo.kryo5.serializers.CachedFields.addField(CachedFields.java:123) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.serializers.CachedFields.rebuild(CachedFields.java:99) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.serializers.FieldSerializer.<init>(FieldSerializer.java:82) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.SerializerFactory$FieldSerializerFactory.newSerializer(SerializerFactory.java:124) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.SerializerFactory$FieldSerializerFactory.newSerializer(SerializerFactory.java:108) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.Kryo.newDefaultSerializer(Kryo.java:469) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.Kryo.getDefaultSerializer(Kryo.java:454) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.util.DefaultClassResolver.registerImplicit(DefaultClassResolver.java:89) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.Kryo.getRegistration(Kryo.java:581) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.Kryo.getSerializer(Kryo.java:601) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.serializers.DefaultArraySerializers$ObjectArraySerializer.read(DefaultArraySerializers.java:346) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.serializers.DefaultArraySerializers$ObjectArraySerializer.read(DefaultArraySerializers.java:299) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.Kryo.readObjectOrNull(Kryo.java:847) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.serializers.ReflectField.read(ReflectField.java:132) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.serializers.FieldSerializer.read(FieldSerializer.java:129) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.Kryo.readObject(Kryo.java:774) ~[utbot-cli-main-2023.8.5040.jar:?]
        at org.utbot.framework.process.kryo.ThrowableSerializer.read(ThrowableSerializer.kt:100) ~[utbot-cli-main-2023.8.5040.jar:?]
        at org.utbot.framework.process.kryo.ThrowableSerializer.read(ThrowableSerializer.kt:20) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.Kryo.readObject(Kryo.java:796) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.serializers.ReflectField.read(ReflectField.java:123) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.serializers.FieldSerializer.read(FieldSerializer.java:129) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.Kryo.readObject(Kryo.java:796) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.serializers.ReflectField.read(ReflectField.java:123) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.serializers.FieldSerializer.read(FieldSerializer.java:129) ~[utbot-cli-main-2023.8.5040.jar:?]
        at com.esotericsoftware.kryo.kryo5.Kryo.readClassAndObject(Kryo.java:877) ~[utbot-cli-main-2023.8.5040.jar:?]
        at org.utbot.framework.process.kryo.KryoHelper.readObject(KryoHelper.kt:72) ~[utbot-cli-main-2023.8.5040.jar:?]
        at org.utbot.instrumentation.ConcreteExecutor$executeAsync$2$1.invokeSuspend(ConcreteExecutor.kt:232) ~[utbot-cli-main-2023.8.5040.jar:?]
        ... 24 more
~~~

~~~java
    ///region Errors report for modifyList
    
    public void testModifyList_errors()
     {
        // Couldn't generate some tests. List of errors:
        // 
        // 30 occurrences of:
        // Default concrete execution failed
        
    }
    ///endregion
~~~

**Environment**

OS - Windows 10 Pro
Windows Powershell 
JDK - 17

**Additional context**

Test generation with utbot-cli-main-2023.6.4550.jar is successful. 
No error report. No exceptions in console.

--- a/utbot-framework-api/src/main/kotlin/org/utbot/framework/process/kryo/KryoHelper.kt
+++ b/utbot-framework-api/src/main/kotlin/org/utbot/framework/process/kryo/KryoHelper.kt
@@ -6,6 +6,7 @@ import com.esotericsoftware.kryo.kryo5.io.Input
 import com.esotericsoftware.kryo.kryo5.io.Output
 import com.esotericsoftware.kryo.kryo5.objenesis.instantiator.ObjectInstantiator
 import com.esotericsoftware.kryo.kryo5.objenesis.strategy.StdInstantiatorStrategy
+import com.esotericsoftware.kryo.kryo5.serializers.JavaSerializer
 import com.esotericsoftware.kryo.kryo5.util.DefaultInstantiatorStrategy
 import com.jetbrains.rd.util.lifetime.Lifetime
 import com.jetbrains.rd.util.lifetime.throwIfNotAlive
@@ -108,6 +109,8 @@ internal class TunedKryo : Kryo() {
         // Kryo cannot (at least, the current used version) deserialize stacktraces that are required for SARIF reports.
         // TODO: JIRA:1492
         addDefaultSerializer(java.lang.Throwable::class.java, ThrowableSerializer())
+
+        addDefaultSerializer(java.lang.StackTraceElement::class.java, JavaSerializer())
 
         val factory = object : SerializerFactory.FieldSerializerFactory() {}
         factory.config.ignoreSyntheticFields = true

